package.json

```diff
@@ -11,31 +11,24 @@
     "build": "tsc",
     "clear": "rm -rf src/*",
     "generate": "node generator.js",
-    "lint": "eslint --ext js,ts,tsx src ./generator.js",
-    "lint:fix": "npm run lint -- --fix",
+    "lint": "eslint --cache --ext js,ts,tsx src ./generator.js",
+    "lint:fix": "npm run lint -- --cache --fix",
     "typecheck": "tsc --noEmit",
     "test": "jest --config jestconfig.json --passWithNoTests",
     "prepare": "npm run build",
     "prepublishOnly": "npm run test && npm run lint",
     "version": "npm run test && npm run lint && git add -A src",
     "postversion": "git push && git push --tags"
   },
-  "keywords": [
-    "material design",
-    "icons",
-    "react"
-  ],
+  "keywords": ["material design", "icons", "react"],
```

Comment:
We should check why this is happening. When I format the document, it is split into several lines again. I don't know what is causing this, though.

Reply:
I'll open another PR for this.